app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,11 @@
+class PostsController < ApplicationController
+  def index
+    # @user = User.find(params[:id])
+    # @posts = @user.posts
```

Comment:
- [ ] Please remove all commented code from your codebase this will greatly help to make your code clean and easy to maintain.

Reply:
well resolved! ✔️